### DISCUSSION TITLE
Renovate fails on a repository, no log

### DISCUSSION THREAD

**Discussion by ysoldak** [renovatebot/renovate#30499] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

We run Renovate as GtitHub app in our org.
Renovate scans of one of repositories that is heavy on npm dependencies started failing from `2024-06-17 14:20:59.600678 +00:00`

No log can be seen for either running or failed runs.

@nabeelsaabna was able to find:
```
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
```
and 
> we did update from 37.393.0 to 37.410.1 on 17 Jun 2024 16:34:03

Original discussion thread:
https://github.com/renovatebot/renovate/discussions/30151#discussioncomment-10031060

Latest failing run id: 01910804-dfc0-71b9-b723-ca5cf5bf2d9c

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by ysoldak** [comment, marked as answer]
It works now for the repository.
Started to be green since 2024-08-06 10:02:58 CEST.
First successful run: 01912646-1218-7f8c-ac14-568e5eb9b9c1

I'd like to monitor it for one more day and then will close the case.

**Reply by nabeelsaabna** [reply]
most likely the Renovate version update to new major v38 :+1:
